### PR TITLE
Whole paper edits

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

**Review by mbobra — APPROVED** [review]

File: sections/9-affiliated.tex [hunk 1]
```diff
@@ -12,9 +12,9 @@ \section{Affiliated Packages}
 \end{itemize}
 Developers can formally apply to become an affiliated package to the lead developer and final approval is required by the SunPy board.
 Packages are re-reviewed on a yearly basis to ensure that they continue to meet the standards.
-All affiliated packages are listed on \url{sunpy.org}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
+All affiliated packages are advertised on the website\footnote{\url{https://sunpy.org/team\#affiliated-packages}}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
```
Comment (nabobalis) [inline sections/9-affiliated.tex:15]:
```suggestion
All affiliated packages are advertised on the website\footnote{\url{https://sunpy.org/team#affiliated-packages}}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
```

File: sections/7-fido.tex [hunk 2]
```diff
@@ -4,16 +4,16 @@ \section{Data search and retrieval}
 One of the most important tasks that must occur before any analysis can take place is to search for and retrieve data.
 A particular science goal may require data from multiple data providers, each of which may have different methods of search and retrieval.
 This heterogeneity increases the effort required by scientists to get the data they need.
-In order to address this issue \sunpypkg provides a single and powerful data search and retrieval interface called \Fido.
+In order to address this issue, the  \package{sunpy.net} subpackage provides a single and powerful data search and retrieval interface called \Fido.
```
Comment (nabobalis) [inline sections/7-fido.tex]:
```suggestion
In order to address this issue, the \package{sunpy.net} subpackage provides a single and powerful data search and retrieval interface called \Fido.
```

File: sections/4-development.tex [hunk 3]
```diff
@@ -12,13 +12,9 @@ \section{Development Model}
     \item All new features must be accompanied with documentation.
     This includes code comments, formal documentation, and gallery examples.
     \item Test code must be provided with coverage reports.
-    \item Finally, all code must be reviewed and accepted by at least two members of the developer community before it is integrated into the codebase.
+    \item Finally, all code must be within scope and be reviewed and accepted by at least two members of the developer community before it is integrated into the codebase.
 \end{enumerate}
-These requirements are imposed on every single contribution.
-
-The \sunpyproj adheres to a schedule of two releases of \sunpypkg per year -- a Long Term Support (LTS) release and a short-support or non-LTS release. 
-LTS releases will be supported for 12 months or until the next LTS release; non-LTS releases will be supported for 6 months or until the next release. Support periods can be extended beyond the requirements here if needed.
```
Comment (nabobalis) [inline sections/4-development.tex:20]: We don't want this somewhere else?

File: sections/11-infrastructure.tex [hunk 4]
```diff
@@ -26,16 +27,16 @@ \subsection{Continuous Integration}
...
-All documentation for \sunpypkg as well as all affiliated packages is written using the commonly-used \code{Sphinx} documentation build system.
+All documentation for \sunpypkg as well as all affiliated packages is written using the commonly-used \code{Sphinx} documentation build system\footnote{\url{http://www.sphinx-doc.org/en/master/}}.
 This system supports using plain text files with a markup language called \code{reStructuredText} (RST).
 The build process converts these files including documentation strings in Python files into HTML, PDF, or LATEX documents.
-We make use of the \code{sphinx-gallery} extension to build a gallery of analysis examples as well as the extension \code{sphinx-automodapi} which generates documentation pages that list all of the available classes, functions, and attributes.
+We make use of the \code{sphinx-gallery}\footnote{\url{https://sphinx-gallery.github.io}} extension to build a gallery of analysis examples as well as the extension \code{sphinx-automodapi}\footnote{\url{https://sphinx-automapi.readthedocs.io}} which generates documentation pages that list all of the available classes, functions, and attributes.
```
Comment (nabobalis) [inline sections/11-infrastructure.tex:41]: Is this too much information? Should we be telling them how we create our API docs?

File: sections/9-affiliated.tex (hunk 1 above)
Comment (ehsteve) [reply sections/9-affiliated.tex:15]: I need to escape this character to get the build to work.

File: sections/7-fido.tex (hunk 2 above)
Comment (ehsteve) [reply sections/7-fido.tex]: fixed.

File: sections/11-infrastructure.tex (hunk 4 above)
Comment (ehsteve) [reply sections/11-infrastructure.tex:41]: This is the same level of detail in the astropy paper and I think it is nice to mention them. I actually which we could do more to give them credit.

File: sections/4-development.tex (hunk 3 above)
Comment (ehsteve) [reply sections/4-development.tex:20]: There is a whole release section. This text is repeated there.

File: sections/9-affiliated.tex (hunk 1 above)
Comment (nabobalis) [reply sections/9-affiliated.tex:15]: Ah good point. I forgot.

**Pull request merged**